articles/azure-vmware/concepts-network-design-considerations.md

```diff
@@ -27,7 +27,7 @@ If you’re using BGP AS-Path Prepend to dedicate a circuit from Azure towards o
 ## Management VMs and default routes from on-premises 
 
 > [!IMPORTANT]
-> Azure VMware Solution Management VMs don't honor a default route from On-Premises.
+> Azure VMware Solution Management VMs does not honor a default route from On-Premises.
```

Comment:
@DP-Miller please change 'does not' to 'do not' to fit the plural that precedes it.